index.html

```diff
@@ -1,4 +1,34 @@
-Raissa 
-I'm Raissa I'm learning to code 
-I'm from Brazil 
-practicing this 
+<!DOCTYPE html>
+<html>
+ <head> 
+    <title>Raissa's</title>
+</head>
+
+ <body>
+
+   <h1>"About" </h1>
```

Comment:
With this "About" header, and the "Experience" one, you don't need to use the quotation marks here in HTML. In Javascript, they're needed to tell the computer that we're using a `string`. But in HTML, everything is basically treated as a string.

If you open your `index.html` file in your Chrome browser, you'll see how the headings currently look:

If it's just your preference on how they look then that's totally fine as well :slightly\_smiling\_face: 

![](https://user-images.githubusercontent.com/9094098/200913445-4b70575c-aa72-4c45-ade4-3b2d7ecf5ef7.png)